react/week3/todolist/src/Components/TodoForm.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import {MdOutlineAddToPhotos} from 'react-icons/md'
+
+const TodoForm = ({ addTodo }) => {
+    const randomId = Math.floor(Math.random() * 10000)
```

Comment:
Bad practice but should work for our purposes now but avoid in bigger projects. Then I can recommend a unique id creator uuid or I see other students using nanoid

Reply:
Oh okay. Will change it and look into it.